tests/testthat/test-check_crpr.R

```diff
@@ -0,0 +1,58 @@
+library(tibble)
+library(dplyr)
+library(lubridate)
+
+adrs <- tribble(
+  ~USUBJID, ~ADTC,        ~AVALC,
+  "1",      "2020-01-01", "PR",
+  "1",      "2020-02-01", "CR",
+  "1",      "2020-02-16", "NE",
+  "1",      "2020-03-01", "CR",
+  "1",      "2020-04-01", "SD",
+  "2",      "2020-02-06", "PR",
+  "2",      "2020-02-16", "CR",
+  "2",      "2020-03-30", "PR",
+) %>%
+  mutate(
+    ADT = ymd(ADTC),
+    STUDYID = "XX1234"
+  )
+
+# signal_crpr ----
+## Test 1: warning is issued ----
+test_that("signal_crpr Test 1: warning is issued", {
+  expect_warning(signal_crpr(
+    adrs,
+    order = vars(ADT)
+  ),
+  paste("Dataset contains CR records followed by PR.",
+    "Run `get_crpr_dataset()` to access the CR records records followed by PR",
```

Comment:
So, just to understand we will need rerun get_crpr_dataset() function to get this data. is it possible to provide the usubjid  with the message itself?

Reply:
@thomas-neitmann as Stefan is away can you answer Amit's question please? Not sure if this is a specific admiral convention to do it this way as I've seen same before with messages to run `get_duplicates_dataset()` for example. I couldn't find anything specifically in the programming strategy on this, but might have missed it.

Reply:
This certainly is an {admiral} convention even though we didn't seem to have documented it properly. When you call `get_crpr_dataset()` you'll easily be able to get the list of all subjects IDs so I don't see a need for printing the IDs as part of the message. Depending on the number of IDs that could actually become rather cluttered.

Reply:
thanks @thomas-neitmann - can you make an admiraldev issue for this? good one to update in the programming strategy after we get past next release

Reply:
@thomas-neitmann @rossfarrugia Thanks its fine we can run the get_crpr_dataset() to get more details. Thanks!!